Extension/src/LanguageServer/configurations.ts

```diff
@@ -624,6 +624,21 @@ export class CppProperties {
         }, () => { }).catch(logAndReturn.undefined);
     }
 
+    public async updateCompilerPathIfSet(path: string): Promise<void> {
+        return this.handleConfigurationEditJSONCommand(() => {
```

Comment:
Doesn't this create a `c_cpp_properties.json` file if one doesn't exist?  We shouldn't be creating a new one, only updating an existing one if it sets the `compilerPath`.

Reply:
I forgot about that. I'll fix it.